src/uct/ib/rdmacm/rdmacm_cm.h

```diff
@@ -50,6 +54,22 @@ typedef struct uct_rdmacm_cm_config {
     double          timeout;
 } uct_rdmacm_cm_config_t;
 
+struct uct_rdmacm_cm_reserved_qpn_blk {
+    ucs_bitmap_t(256)       *bmp;
+    uint32_t                first_qpn;
+    ucs_list_link_t         entry;
+    unsigned int            next_avail_slot;
```

Comment:
according to usage this is `offset` from `first_qpn`, right? then please rename and put these fields together

Reply:
@evgeny-leksikov Renamed and reordered.